src/main/scala/com/touk/hiring_task/multiplex/commons/MultiplexException.scala

```diff
@@ -0,0 +1,26 @@
+package com.touk.hiring_task.multiplex.commons
+
+trait MultiplexException extends RuntimeException
+
+trait ValidationException extends MultiplexException
+
+case object InternalServerError extends MultiplexException
+case object ProjectionNotFoundException extends MultiplexException
+case object NoVacanciesForMovieException extends MultiplexException
+case object SelectedPlaceUnavailableException extends MultiplexException
+case object SelectedPlaceNotAvailableException extends MultiplexException
+
+case object InvalidUsernameException extends ValidationException
+case object InvalidStartProjectionDateException extends ValidationException
```

Comment:
I see that you declared the validation errors as an exception, but AFAIS you don't throw them but return in Either.Left. Why did you decide to model it in this way?